x-pack/plugins/osquery/public/action_results/translations.ts

```diff
@@ -20,3 +20,24 @@ export const FAIL_ACTION_RESULTS = i18n.translate(
     defaultMessage: `Failed to fetch action results`,
   }
 );
+
+const platinumLicenseRequired = 'At least Platinum license is required to use Response Actions.';
+const parametersNotFound =
+  "This query hasn't been called due to parameter used and its value not found in the alert.";
```

Comment:
Is it possible for these to be placed in a area where both the server-side and UI side can use?

Keeping your version of these messages here feels fragile and prone to bugs if the server side errors are ever changed (by us, or a "future" us 😄 ).